### PR TITLE
Handler for analysing golang packages

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: f8a_jobs/handlers/golang_popular_analyses.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    def _popular_packages(self):
+        """Schedule analyses of popular packages in golang."""
+        endpoint = self._URL + '?action=tops&len=100'
+
+        response = requests.get(endpoint)
+        response.raise_for_status()
+
+        packages_seen = 0
+        scheduled = 0
+        for top_category in response.json():
```
Comment (jpopelka) [inline f8a_jobs/handlers/golang_popular_analyses.py]: I think we are not interested in the `"Sites"` category as it does not contain packages, just number of packages in each site.

Comment (fridex) [reply f8a_jobs/handlers/golang_popular_analyses.py]: Nice catch! Updated, thanks.

**Comment by jpopelka** [comment]
Remove `WIP` once you're ready for merge.

**Comment by fridex** [comment]
> Remove WIP once you're ready for merge.

I think this will need some additional work if we would like to analyse versions as well, but for now we should be fine with merging..

**Pull request merged**